RELEASES.md

```diff
@@ -1,29 +1,54 @@
 # Creating a new release (guidelines for repo-admins)
 
-After review of a protocol-branch and before each release the following steps are necessary:
+After review of a protocol-branch and before each release the following steps will be run automatically and may require action:
 
-1.  with the protocol-branch checked-out:
+1.  A GitHub Action will run when a pull request is made to the main branch.
 
-    1.  bump the version number in the `index.Rmd` yaml section from `yyyy.nn.dev` to `YYYY.NN`.
-        Some examples:
+    1.  This action will automatically run these checks:
 
-        -   a new protocol added in 2020 that was released in 2020 (fifth release in that year): `2020.00.dev` to `2020.05`
-        -   an update of the `2020.05` protocol in 2021 (first release in that year): `2020.05.dev` to `2021.01`
-        -   an update of the `2020.02` version of a protocol in the same year (fourth release in that year): `2020.02.dev` to `2020.04`
+        1.  `protocolhelper::check_frontmatter()`
 
-    2.  update the `.zenodo.json` file:
+        2.  `protocolhelper::check_structure()`
 
-        -   add new authorships with the aid of `protocolhelper:::update_zenodo()`
+    2.  Check any error messages resulting from these checks and address them.
+        Repeat this step until all checks succeeded.
+        In principle, these errors should be addressed by the protocol contributor, but admins can help if needed.
 
-    3.  update the repo `NEWS.md` file
+2.  A second GitHub Action will be run when a pull request to be merged into the main branch *upon approval*.
 
-2.  merge the branch to the main and add general and specific tags (see [release model](README.md#release-model)):
+    1.  This action will automatically update:
 
-    1.  general tag: `protocols-YYYY.NN`
-    2.  specific tag: `protocol-code-YYYY.NN`
+        1.  the repo-wide `NEWS.md` file with `protocolhelper:::update_news_release()`
 
-3.  Check if continuous integration proceeded without problems
+        2.  the `.zenodo.json` file with `protocolhelper:::update_zenodo()`
 
-4.  Make a GitHub release from the general tag
+    1..
+    Check if the second Github Action ran without problems.
+    If needed, admins should address any problems.
+    Note that during these automatic updates, the two commits will be automatically pushed to the remote.
+    This should not pose a problem for local clones as these changes can be automatically included and will not cause conflicts (i.e. `git pull origin main` with a local protocol branch checked out)
```

Comment:
```suggestion
    To add these commits to your local clone, press pull in the RStudio Git Pane.
```
I suppose also repo-admins don't need this long description to remember they have to pull?  ;-)